### ISSUE TITLE
Update code standards for API / solutions code

### ISSUE THREAD

**Issue by colleenskemp** [ProjectDrawdown/solutions#240]
@sutjin and @brodavi  - Will you contribute to the acceptance criteria for this ticket?

**Comment by colleenskemp** [comment]
What are we asking for here? 

Style guides? 

@sutjin - Can you provide direction about what is anticipated here?

**Comment by sutjin** [comment]
@colleenskemp Created a code guideline for the Service project including a few more documentation. I added a pylint enforcer that I have modified to fit the standard that we would want.

If other developers are coding with an IDE, the pylint will try to enforce the rules via the IDE. I also tried to add a `pre-commit` hook where it will try to enforce the rules before they can open a pull request.

However `pre-commit` are on a per machine base and will not be automatically applied when someone pulls in our project initially. That solution will be done via `pre-receive`, which I don't want to enable yet until we at least get our end of the clean up done, otherwise to PR will be accepted

global-research-platform/pull/51

As mentioned with @eethann , further documentation of codes will be done on an "as-we-go" basis as we touched on the undocumented code
